hooks.go

```diff
@@ -0,0 +1,61 @@
+package logging
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+
+	"github.com/bshuster-repo/logrus-logstash-hook"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/gemnasium/logrus-graylog-hook.v2"
+)
+
+func (c LogConfig) initLogstashHook(h LogHook) error {
+	if err := c.validateRequiredHookSettings(h, []string{"network"}); err != nil {
+		return err
+	}
+	network, _ := h.Settings["network"]
```

Comment:
It should be either `tcp` or `udp`, any other value will give you an error in `net.Dial()`, so even if it is not set - empty value is fine, error will be thrown later.